grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -1120,8 +1120,10 @@ class StockMovementService {
 
     void removeRequisitionItem(RequisitionItem requisitionItem) {
         Requisition requisition = requisitionItem.requisition
-        removeShipmentItemsForModifiedRequisitionItem(requisitionItem)
         requisitionItem.undoChanges()
+        requisitionItem.save(flush: true)
```

Comment:
@DominikaHatala Are you sure the save/flush is necessary?

Reply:
@jmiranda Yes, in the ticket I've let you know about the hibernate error that occured before adding save/flush.